internal/environments/metadata.go

```diff
@@ -0,0 +1,13 @@
+package environments
+
+type Metadata struct {
+	Cluster     string `mapstructure:"ha-cluster,omitempty"`
+	Environment string `mapstructure:"sap-environment,omitempty"`
+	Landscape   string `mapstructure:"sap-landscape,omitempty"`
+	SAPSystem   string `mapstructure:"sap-system,omitempty"`
+}
+
+func NewMetadata() Metadata {
+	var metadata = Metadata{}
+	return metadata
+}
```

Comment:
This function doesn't do any initialization logic: calling `new(Metadata)` or assigning `Metadata{}` to a variable would be identical. Maybe we can just remove it?